docs/generated/manifests/menus.json

```diff
@@ -3521,6 +3521,23 @@
         "id": "showcase",
         "isExternal": false,
         "children": [
+          {
+            "name": "Add a New Project",
```

Comment:
For some reason I cannot collapse this in the side-bar.

Also a question that came to my mind: how do these "Add a New Project" differ from the example repos? Couldn't this be a more elaborate page of the examples ones, like https://nx-dev-git-fork-isaacplmann-docs-add-express-nrwl.vercel.app/showcase/example-repos/apollo-react?

https://github.com/nrwl/nx-recipes/tree/main/lit is also in the `nx-recipes`, so would that also be shown in the "Example Repos" section?

Reply:
These should be more polished and are all similar in style.  The example repos could be anything.  They could be showing how multiple technologies can be used together.  They could be showing how a particular complex set up works.

All the "Add a New Project" recipes will be the same style and show people just what they need to know to get started with a single new stack.

Reply:
Also, it looks like you can't collapse the section of a page you're currently looking at.  I don't know if that's intentional or not.

Reply:
> These should be more polished and are all similar in style. The example repos could be anything. They could be showing how multiple technologies can be used together. They could be showing how a particular complex set up works.

I'm just skeptical whether to have both vs just going with "Examples" and have the mixed there, where some could be more elaborate compared to others. Because in this case we would need to have the Lit also in the "Example Repos", or at least I'd expect to see it there.

> Also, it looks like you can't collapse the section of a page you're currently looking at. I don't know if that's intentional or not.

Could be..

Reply:
ok, I moved the recipe under "Example Repos". It seems weird to have "Showcase" with only one item under it that is also a group, but it isn't a big deal.  Maybe "Example Repos" should be a top level item then.